notebooks/MAST/TESS/interm_tesscut_dss_overlay/tesscut_dss_overlay.ipynb

```diff
@@ -0,0 +1,633 @@
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"overlay_images\"></a>\n",
+    "## Overlay the TessCut Image\n",
+    "To overlay the TESS Cutout against the DSS, we need to reproject the WCS of the TESS cutout onto the WCS of the DSS image, and the [reproject](https://reproject.readthedocs.io/en/stable/index.html) package.  We use the [reproject_interp](https://reproject.readthedocs.io/en/stable/api/reproject.reproject_interp.html#reproject.reproject_interp) method.  See [here](https://reproject.readthedocs.io/en/stable/celestial.html) for more details. \n",
+    "\n",
+    "``reproject_interp`` takes two basic inputs.  \n",
+    "- input data+WCS we want to project from\n",
+    "- output WCS we want to project onto.  \n",
+    "\n",
+    "We want to project our TESS cutout and its WCS into the coordinate frame of the DSS image, using the DSS WCS.  Our input is a tuple of the custom TESS cutout data and wcs, i.e. ``(data, wcs)``.  The desired coordinateframe is the WCS we want to project onto, i.e. ``dss_wcs``.  We also need to provide the shape of the DSS data, with `shape_out`.  Finally we need to provide an interpolation method.  We use a ``nearest-neighbor`` interpolation, which is the lowest order possible.  \n",
+    "\n",
+    "``reproject_interp`` will take the input 11x11 pixel TESS cutout data, convert it to world coordinates using the TESS WCS, interpolate it into the shape of the DSS image (229x229) using the DSS WCS, and convert the data into DSS pixel coordinates.  \n",
+    "\n",
+    "The final output is the reprojected TESS cutout data (``reproj_tesscut``) and the overlapping footprint of the input data in the output data (``footprint``).  \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# reproject the tesscut onto dss\n",
+    "reproj_tesscut, footprint = reproject_interp((data, wcs,), dss_wcs, shape_out=dss_data.shape, order='nearest-neighbor')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Once reprojected, we can properly overly the DSS and TESSCut images.  We can add a slider for the opacity of the overlay using ipywidgets.  First we make a `create_plot` function which accepts all the inputs we need to display the DSS image, and overlay the TESS cutout image and TIC objects.  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# create the function to overlay dss + tesscut + tic catalog\n",
+    "def create_plot(data, array, alpha, norm, wcs, ra, dec):\n",
+    "    ''' create an interactive matplotlib image plot\n",
+    "    \n",
+    "    Parameters:\n",
+    "        data (ndarray):\n",
+    "            The background image data to show\n",
+    "        array (ndarray):\n",
+    "            The foreground image data to overplot. \n",
+    "        alpha (tuple):\n",
```

Comment:
Possibly more clear to have the input arguments be something like `background_img`, and `foreground_img` rather than `data` and `array`.